### PR TITLE
Implemented Sessions page

### PR DESCRIPTION
**Pull request by vedangimittal**
- Implemented Sessions page
- Jira story: https://jsw.ibm.com/browse/PFEBMC-2397

### CODE REVIEW

**Comment by vedangimittal** [comment]
- Checkbox - row select functionality is still broken
- Changes required in TableRowAction.vue file

File: src/locales/en-US.json [hunk 1]
```diff
@@ -1810,7 +1810,7 @@
       "message": "For users with an HMC attached, if you try to disconnect it from this page and the HMC is active, it will attempt to reconnect. The proper way to disconnect a working HMC is to disconnect it from the HMC itself."
     },
     "modal": {
-      "disconnectMessage": "Are you sure you want to disconnect %{count} session? This action cannot be undone. | Are you sure you want to disconnect %{count} sessions? This action cannot be undone.",
+      "disconnectMessage": "Are you sure you want to disconnect 1 session? This action cannot be undone. | Are you sure you want to disconnect %{count} sessions? This action cannot be undone.",
```
Comment (Nikhil-Ashoka) [inline src/locales/en-US.json]: @vedangimittal, This change is already present in other languages. So, I would prefer us using the same Message and avoid rework for this in translation files.
Eg: https://github.ibm.com/openbmc/webui-vue/blob/1060-ghe/src/locales/de.json

File: src/views/SecurityAndAccess/Sessions/Sessions.vue [hunk 2]
```diff
@@ -11,72 +11,81 @@
...
+              @change="
+                toggleSelectRow(
+                  tableSessionsRef,
+                  row.index,
+                  sessionsStore.allConnections[row.index].isSelected,
+                  row.item,
+                )
+              "
+            ></BFormCheckbox>
+            <!-- @update:modelValue="toggleOne" -->
```
Comment (Nikhil-Ashoka) [inline src/views/SecurityAndAccess/Sessions/Sessions.vue]: Remove unused code.

File: src/views/SecurityAndAccess/Sessions/Sessions.vue [hunk 3]
```diff
@@ -92,214 +101,232 @@
...
+          :per-page="itemPerPage"
           :total-rows="getTotalRowCount(filteredRows)"
           aria-controls="table-session-logs"
         />
-      </b-col>
-    </b-row>
-  </b-container>
+      </BCol>
+    </BRow>
+    <BModal
```
Comment (Nikhil-Ashoka) [inline src/views/SecurityAndAccess/Sessions/Sessions.vue:134]: Here, I think we can use only 1 modal and optimize?

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]

File: src/views/SecurityAndAccess/Sessions/Sessions.vue [hunk 4]
```diff
@@ -11,72 +11,80 @@
...
           </div>
         </alert>
-      </b-col>
-    </b-row>
-    <b-row class="align-items-end">
-      <b-col sm="6" md="5" xl="4">
+      </BCol>
+    </BRow>
+    <BRow class="align-items-end">
+      <BCol sm="6" md="5" xl="4" class="searchStyle">
```
Comment (Nikhil-Ashoka) [inline src/views/SecurityAndAccess/Sessions/Sessions.vue:17]: SearchStyle is not used. Hence, styling is not proper.
Screenshot 2024-10-07 at 1 54 45 PM

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]

**Review by Nikhil-Ashoka — APPROVED** [review]

**Review by nabilpandiyalayil — APPROVED** [review]

**Pull request merged**